products/firefox_desktop/adding_nightly.md

```diff
@@ -55,21 +44,43 @@ $ hg pull -u
 $ hg up central
 ```
 
-The file to modify is in `browser/locales/all-locales`, open it with your text editor of choice.
+The first file to modify is `browser/locales/all-locales`, open it with your text editor of choice.
 
 ```BASH
 $ atom browser/locales/all-locales
 ```
 
 And add the new locale to the list. With Atom and the Sort Lines package installed, you can press `F5` to make sure that the list is in **alphabetical order**.
 
-After you’ve finished editing the file, check the status of the repository, and the diff.
+The second file to modify is `browser/locales/l10n.toml`. This is the beginning of the file:
+
+```
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+basepath = "../.."
+
+locales = [
+    "ach",
+    "af",
+    "an",
+    "ar",
+    "as",
+...
+```
+
+Identify the `locales` section, and add the new locale code between parenthesis, followed by a comma. As before, you can use Atom to make sure the list is in alphabetical order (make sure to select only the lines with actual locale codes before pressing `F5`).
```

Comment:
Hmm "parenthesis" should be "quotes" (or "quotation marks", not sure anymore of the English term) here I believe :)

Reply:
oh boy, those are definitely not parenthesis…

"double quotes"?